<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>V.C.打包工具</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="cmdOutptEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="pakFileNameBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="selExeFileBtn">
        <property name="text">
         <string>打包！</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openPakgDirBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>打开文件夹</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
